### PR TITLE
Add dutch translation

### PR DESCRIPTION
**Pull request by LeandroMarceddu**

### CODE REVIEW

**Comment by shinuza** [comment]
Thank you for your contribution. We will be looking into it.

File: src/store/config/index.ts [hunk 1]
```diff
@@ -18,7 +18,8 @@ export const defaultState = (): ConfigState => {
       locales: [
         { name: 'English', code: 'en' },
         { name: 'Français', code: 'fr' },
-        { name: 'Chinese', code: 'cn' }
+        { name: 'Chinese', code: 'cn' },
+		{ name: 'Nederlands', code: 'nl' }
```
Comment (shinuza) [inline src/store/config/index.ts:22]: I take it that your editor didn't read the `.editorconfig` file, can you check if this is not a tab and/or if you have the correct amount of spaces in there?

**Review by shinuza — CHANGES_REQUESTED** [review]

File: src/store/config/index.ts (hunk 1 above)
Comment (LeandroMarceddu) [reply src/store/config/index.ts]: Fixed

Comment (shinuza) [reply src/store/config/index.ts]: Thanks

**Comment by cadriel** [comment]
Merged, and thank you!

**Pull request merged**